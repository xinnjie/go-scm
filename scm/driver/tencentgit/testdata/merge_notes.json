[
  {
    "id": 1383976,
    "parent_id": null,
    "body": "test comment",
    "attachment": null,
    "author": {
      "id": 176405,
      "username": "xinnjie",
      "web_url": "https://git.code.tencent.com/u/xinnjie",
      "name": "xinnjie",
      "state": "active",
      "avatar_url": "https://git.code.tencent.com/uploads/user/avatar/176405/7e72ab97e19e430287c34c54af3bf7e1."
    },
    "created_at": "2021-08-24T11:41:23+0000",
    "system": false,
    "line_code": null,
    "file_path": null,
    "comments": [
      {
        "comment": 2,
        "created_at": "2021-08-24T12:13:29+0000",
        "author": {
          "id": 176405,
          "username": "xinnjie",
          "web_url": "https://git.code.tencent.com/u/xinnjie",
          "name": "xinnjie",
          "state": "active",
          "avatar_url": "https://git.code.tencent.com/uploads/user/avatar/176405/7e72ab97e19e430287c34c54af3bf7e1."
        }
      }
    ]
  },
  {
    "id": 1384041,
    "parent_id": null,
    "body": "comments in files",
    "attachment": null,
    "author": {
      "id": 176405,
      "username": "xinnjie",
      "web_url": "https://git.code.tencent.com/u/xinnjie",
      "name": "xinnjie",
      "state": "active",
      "avatar_url": "https://git.code.tencent.com/uploads/user/avatar/176405/7e72ab97e19e430287c34c54af3bf7e1."
    },
    "created_at": "2021-08-24T12:12:53+0000",
    "system": false,
    "line_code": "2351e25adcb04ae078a7046ab575d5107ff7b457_0_1",
    "file_path": "close-pr-file.md",
    "comments": []
  },
  {
    "id": 1384048,
    "parent_id": 1384041,
    "body": "reply comment in files",
    "attachment": null,
    "author": {
      "id": 176405,
      "username": "xinnjie",
      "web_url": "https://git.code.tencent.com/u/xinnjie",
      "name": "xinnjie",
      "state": "active",
      "avatar_url": "https://git.code.tencent.com/uploads/user/avatar/176405/7e72ab97e19e430287c34c54af3bf7e1."
    },
    "created_at": "2021-08-24T12:14:07+0000",
    "system": false,
    "line_code": "2351e25adcb04ae078a7046ab575d5107ff7b457_0_1",
    "file_path": "close-pr-file.md",
    "comments": []
  }
]