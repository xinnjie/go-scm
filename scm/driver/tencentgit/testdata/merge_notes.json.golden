[
  {
    "ID": 1383976,
    "Body": "test comment",
    "Author": {
      "ID": 0,
      "Login": "xinnjie",
      "Name": "xinnjie",
      "Email": "",
      "Avatar": "https://git.code.tencent.com/uploads/user/avatar/176405/7e72ab97e19e430287c34c54af3bf7e1.",
      "Link": "",
      "Created": "0001-01-01T00:00:00Z",
      "Updated": "0001-01-01T00:00:00Z"
    },
    "Link": "",
    "Version": 0,
    "Created": "2021-08-24T11:41:23Z",
    "Updated": "2021-08-24T11:41:23Z"
  },
  {
    "ID": 1384041,
    "Body": "comments in files",
    "Author": {
      "ID": 0,
      "Login": "xinnjie",
      "Name": "xinnjie",
      "Email": "",
      "Avatar": "https://git.code.tencent.com/uploads/user/avatar/176405/7e72ab97e19e430287c34c54af3bf7e1.",
      "Link": "",
      "Created": "0001-01-01T00:00:00Z",
      "Updated": "0001-01-01T00:00:00Z"
    },
    "Link": "",
    "Version": 0,
    "Created": "2021-08-24T12:12:53Z",
    "Updated": "2021-08-24T12:12:53Z"
  },
  {
    "ID": 1384048,
    "Body": "reply comment in files",
    "Author": {
      "ID": 0,
      "Login": "xinnjie",
      "Name": "xinnjie",
      "Email": "",
      "Avatar": "https://git.code.tencent.com/uploads/user/avatar/176405/7e72ab97e19e430287c34c54af3bf7e1.",
      "Link": "",
      "Created": "0001-01-01T00:00:00Z",
      "Updated": "0001-01-01T00:00:00Z"
    },
    "Link": "",
    "Version": 0,
    "Created": "2021-08-24T12:14:07Z",
    "Updated": "2021-08-24T12:14:07Z"
  }
]
